scss/civicrm/common/_bootstrap-calendar.scss

```diff
@@ -1,124 +1,122 @@
-.page-civicrm {
-  .ui-datepicker {
-    z-index: 9999 !important;
-    min-width: 160px;
-    width: auto !important;
-    padding: 10px !important;
-    border: 1px solid $datepicker-border-color !important;
-    border-radius: 0;
-    box-shadow: $box-shadow !important;
-
-    .ui-datepicker-header {
-      .ui-datepicker-prev,
-      .ui-datepicker-next {
-        border-radius: 2px;
-        cursor: pointer;
-        text-align: center;
-        width: 35px;
+.ui-datepicker {
```

Comment:
i just realized: why is this partial called `_bootstrap-calendar` when it contains the datepicker style? And what's the difference between this partial and `common/_datepickers.scss` then?

Reply:
These files style a date-picker I will be more appropriate to keep the styles in single partial.

Reply:
@bmnepali I assume @AkA84 agreed with your comment? Could you confirm please?